<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nerv</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>1181</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Next">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Prev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>799</width>
       <height>549</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>471</width>
      <height>551</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>469</width>
       <height>549</height>
      </rect>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
